gluster_brick_provisioning.adoc

```diff
@@ -0,0 +1,41 @@
+// vim: tw=79
+= Gluster brick provisioning
+
+This document describes some of the conventions followed when tendrl provisions
+bricks for gluster volumes. Also it provides guidelines that user has to follow
+if he creates brick manually and tendrl has to pick that.
+
+== Conventions followed by tendrl for Gluster brick creation
+
+ * User will have to provide just the brick name, disk and node id.
+
+ * Mount point for the brick will be created at "/tendrl_gluster_bricks/<brick_name>_mount"
+
+ * The actual brick path will be "/tendrl_gluster_bricks/<brick_name>_mount/<brick_name>"
```

Comment:
When is `/tendrl_gluster_bricks/<brick_name>_mount/<brick_name>` created? After `GlusterCreateBrick` job finishes I have created only directory  `/tendrl_gluster_bricks/<brick_name>_mount/`.